src/rules/order.js

```diff
@@ -243,28 +247,24 @@ function makeOutOfOrderReport(context, imported) {
   reportOutOfOrder(context, imported, outOfOrder, 'before')
 }
 
-function importsSorterAsc(importA, importB) {
-  if (importA < importB) {
-    return -1
-  }
-
-  if (importA > importB) {
-    return 1
-  }
+function getSorter(ascending) {
+  let multiplier = (ascending ? 1 : -1);
 
-  return 0
-}
+  return function importsSorter(importA, importB) {
+    let result
 
-function importsSorterDesc(importA, importB) {
-  if (importA < importB) {
-    return 1
-  }
+    if ((importA < importB) || importB === null) {
+      result = -1
+    }
+    else if ((importA > importB) || importA === null) {
+      result = 1
+    }
+    else {
```

Comment:
```suggestion
    } else if ((importA > importB) || importA === null) {
      result = 1
    } else {
```
always cuddle your elses :-)